components/sup/doc/http_gateway_services_schema.json

```diff
@@ -19,30 +19,71 @@
       },
       "type": "object"
     },
-    "package_identifier": {
-      "description": "A Habitat package",
+    "sys_extended": {
+      "description": "System information",
```

Comment:
Adding a `$comment` key (the `$` is important) here explaining the difference between `sys_basic` and `sys_extended` would be useful to help guide #5692.

Reply:
Will do